packages/governance/test/unitTests/test-multiPluralityBallotCount.js

```diff
@@ -0,0 +1,291 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { makeStoredPublishKit } from '@agoric/notifier';
+import {
+  eventLoopIteration,
+  makeFakeMarshaller,
+} from '@agoric/notifier/tools/testSupports.js';
+import { makeMockChainStorageRoot } from '@agoric/vats/tools/storage-test-utils.js';
+import { makeHandle } from '@agoric/zoe/src/makeHandle.js';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { E } from '@endo/eventual-send';
+import {
+  ChoiceMethod,
+  coerceQuestionSpec,
+  ElectionType,
+  QuorumRule,
+} from '../../src/index.js';
+import { makeMultiPluralityVoteCounter } from '../../src/multiPluralityVoteCounter.js';
+
+const SIMPLE_ISSUE = harden({
+  text: 'Fish or Beef or Pork or Chicken or Duck?',
+});
+const FISH = harden({ text: 'Fish' });
+const BEEF = harden({ text: 'Beef' });
+const PORK = harden({ text: 'Pork' });
+const CHICKEN = harden({ text: 'Chicken' });
+const DUCK = harden({ text: 'Duck' });
+
+const FAKE_CLOSING_RULE = {
+  timer: buildManualTimer(console.log),
+  deadline: 3n,
+};
+
+const FAKE_COUNTER_INSTANCE = makeHandle('Instance');
+
+function makePublisherFromFakes() {
+  const storageRoot = makeMockChainStorageRoot();
+  const publishKit = makeStoredPublishKit(storageRoot, makeFakeMarshaller());
+  return { publisher: publishKit.publisher, storageRoot };
+}
+
+test('multi plurality contested', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK, CHICKEN, DUCK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
```

Comment:
I want to see some tests where
 * maxChoices is > 1, and some voters specify multiple candidates
 * There are 5 candidates, and their final vote totals are like [8,5,5,5,5,3], so we can see that the tiebreaker chooses 5 winners.
 * There are 6 candidates, 4 possible winners, but voters only specify 3 of the candidates, so the list of winners is short.